test/controllers/SubmitFormControllerSpec.scala

```diff
@@ -20,64 +20,78 @@ import java.time.LocalDate
 
 import base.BaseSpec
 import connectors.httpParsers.ResponseHttpParsers.HttpGetResult
-import mocks.{MockVatObligationsService, MockVatSubscriptionService}
-import models.{CustomerDetails, VatObligation, VatObligations}
+import mocks.{MockAuth, MockVatObligationsService, MockVatSubscriptionService}
 import models.errors.UnexpectedJsonFormat
+import models.{CustomerDetails, VatObligation, VatObligations}
 import play.api.http.Status
 import play.api.test.Helpers._
 
 import scala.concurrent.Future
 
-class SubmitFormControllerSpec extends BaseSpec with MockVatSubscriptionService with MockVatObligationsService {
+class SubmitFormControllerSpec extends BaseSpec with MockVatSubscriptionService with MockVatObligationsService with MockAuth {
```

Comment:
You don't need to extend from BaseSpec anymore since this is included with MockAuth